### PR TITLE
Add a screenshot of the webapp in the README

### LINKED ISSUE #65: [FEAT] Screenshot in the README
# Feature Request

## Describe the solution you'd like
Add a screenshot in the README file for the users to know how simple and effective the website actually is

### PR DESCRIPTION
**Pull request by sachinh19**
Signed-off-by: sachinh19 <sachinhaldavanekar@gmail.com>

# Pull Request

I read the guidelines, the guidelines mentioned that I needed to request on the issue that I would like to work on this and I commented on it. I also read a comment there mentioning that the first comprehensive PR would be reviewed and merged, so I went ahead with creating this.

## Description:

Adds a screenshot of the web app's usage in the README file.

## Related Issue:

Closes #65

### CODE REVIEW

**Comment by adityaraute** [comment]
Why not paste the image directly?

**Comment by nhcarrigan** [comment]
> Why not paste the image directly?

That works if you're editing the file directly on GitHub, but doesn't work locally. I much prefer this approach, actually, as it means replacing the image can be done by modifying the image file directly.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ A basic application that generates a chart to display an event's start/end time
 
 This page is currently deployed. [View the live website.](https://www.nhcarrigan.com/timezone-chart)
 
+![](assets/sample.png)
+
```
Comment (nhcarrigan) [inline README.md:10]:
```suggestion
![A screenshot of the website, depicting a form with date-time inputs, a dropdown with timezone selection, and a submit button.](assets/sample.png)

```

We should add alt-text to the image for accessibility purposes.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by adityaraute — DISMISSED** [review]

**Review by nhcarrigan — APPROVED** [review]
Looks good to me. I've given this a rebase - will merge once the CI passes.

**Comment by nhcarrigan** [comment]
Congrats on your first pull request (PR).

Thank you for your contribution to the page. We are happy to accept these changes and look forward to future contributions.

**Pull request merged**